numpy/lib/tests/test_arraypad.py

```diff
@@ -1009,6 +1009,21 @@ def test_unicode_mode(self):
         assert_array_equal(a, b)
 
 
+class TestObjectInput(object):
+    def test_object_input(self):
+        # Regression test for issue gh-11395.
+        a = np.full((4, 3), None)
+        pad_amt = ((2, 3), (3, 2))
+        b = np.full((9, 8), None)
+        modes = ['edge',
+                 'symmetric',
+                 'reflect',
+                 'wrap',
+                 ]
```

Comment:
Deeply weird style, but I can't fault you for copying what was already in this file